src/main/java/com/okta/tools/OktaAwsConfig.java

```diff
@@ -56,7 +61,16 @@ private static String getEnvOrConfig(Properties properties, String propertyName)
                 envValue : properties.getProperty(propertyName);
     }
 
+    private static String getProfile(String profileFromCmdLine, String profileFromEnvOrConfig) {
+        return profileFromCmdLine != null ?
+                profileFromCmdLine : profileFromEnvOrConfig;
+    }
+
     private static Integer getStsDurationOrDefault(String stsDuration) {
         return (stsDuration == null) ? 3600 : Integer.parseInt(stsDuration);
     }
+
+    private static String getAwsRegionOrDefault(String region) {
+        return (region == null) ? "us-east-1" : region;
+    }
```

Comment:
👍 it's good to make that default explicit. It causes some confusion for sure.

You can work around this in the current release by setting region after login with aws configure, but it's awkward. This is a big improvement for multi-region use.